acceptance-tests/tests/src/test/resources/jsonrpc/eth/getBlockBy/test-cases/07_get_first_shanghai_block.json

```diff
@@ -0,0 +1,35 @@
+{
+  "request": {
+    "jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x2", false],"id":1
+  },
+  "response": {
+    "jsonrpc": "2.0",
+    "id": 1,
+    "result" : {
+      "number" : "0x2",
+      "hash" : "0x7c2a09bedc85268606805e6df579e758af8d832ef9826927ce4b4b14452a32cf",
+      "mixHash" : "0x0000000000000000000000000000000000000000000000000000000000000000",
+      "parentHash" : "0x3559e851470f6e7bbed1db474980683e8c315bfce99b2a6ef47c057c04de7858",
+      "nonce" : "0x0000000000000000",
+      "sha3Uncles" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+      "logsBloom" : "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      "transactionsRoot" : "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+      "stateRoot" : "0xca3149fa9e37db08d1cd49c9061db1002ef1cd58db2210f2115c8c989b2bdf45",
+      "receiptsRoot" : "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+      "miner" : "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b",
+      "difficulty" : "0x0",
+      "totalDifficulty" : "0x400000000",
+      "extraData" : "0x",
+      "baseFeePerGas" : "0x7",
+      "size" : "0x21c",
+      "gasLimit" : "0x1c9c380",
+      "gasUsed" : "0x0",
+      "timestamp" : "0x10",
+      "uncles" : [ ],
+      "transactions" : [ ],
+      "withdrawalsRoot" : "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+      "withdrawals": [ ]
```

Comment:
Can we also test this block with the empty withdrawals case using eth_getBlockByHash

Reply:
yeh sure. I know the wiring is the same code underneath so took a short cut, but maybe I should go whole hog and test last paris block too?

Reply:
don't we need to test the last Paris block, just testing the shanghai block would be enough